app/routes/order/international_order.js

```diff
@@ -0,0 +1,19 @@
+import Ember from "ember";
+import AuthorizeRoute from "./../authorize";
+
+export default AuthorizeRoute.extend({
+  orderService: Ember.inject.service(),
+
+  async model() {
+    return {
+      shipmentSubsequentCode: await this.get(
```

Comment:
I guess you can completely remove the model here
Reason is that ur model method is actually not referring any model.
And you can move this to `setupController` completely

Remove this and make setupController as
```javascript
async setupController(controller, model) {
    this._super(controller, model);
     const code  = await this.get("orderService").fetchShipmentorCarryoutCode("Shipment");
   controller.set("shipmentorCarryoutCode", code);
 }
```